src/jarabe/model/buddy.py

```diff
@@ -113,16 +115,38 @@ def __init__(self):
                 dbus_interface=dbus.BUS_DAEMON_IFACE):
             if service.startswith(CONNECTION + '.'):
                 path = '/%s' % service.replace('.', '/')
-                Connection(service, path, bus,
-                           ready_handler=self.__connection_ready_cb)
+                conn_proxy = bus.get_object(service, path)
+                self._prepare_conn_cb(path, conn_proxy)
+
+    def _prepare_conn_cb(self, object_path, conn_proxy):
```

Comment:
Function name does not use our style; it is not a callback, so the name should not end in `_cb`.

Reply:
Thanks for noticing